### DISCUSSION TITLE
zepeto studio 업로드 이상

### DISCUSSION THREAD

**Discussion by Astoration** [naverz/zepeto-studio-kor#820] [5. Q&A]
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/13393367/170953766-7e6edf40-f6e1-4461-949e-99de60a8a90d.png">
현재 이상태로 무한 반복중이여서 취소도, 삭제도, 변경도 되지 않습니다.
다른계정으로 시도해도 동일합니다 뭔가 문제가 있는걸까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 불편을 드려 죄송합니다.
관련 내용 수정하여 지금은 정상적으로 업로드 됨을 확인하였습니다.
감사합니다.
